plugins/browser-plugin-snowplow-ecommerce/src/api.ts

```diff
@@ -240,6 +242,28 @@ export function trackRefund(
   });
 }
 
+/**
+ * Track a transaction error event
+ *
+ * @param transactionError - The transaction error information
+ * @param trackers - The tracker identifiers which the event will be sent to
+ */
+export function trackTransactionError(
+  transactionError: TransactionError & CommonEcommerceEventProperties,
+  trackers: Array<string> = Object.keys(_trackers)
+) {
+  const { context = [], timestamp, transaction, ...transactionErrorAttributes } = transactionError;
+  context.push({
+    schema: TRANSACTION_SCHEMA,
+    data: { ...transaction },
+  });
+  context.push({ schema: TRANSACTION_ERROR_SCHEMA, data: { ...transactionErrorAttributes } });
```

Comment:
Nit: Probably don't need to repack transactionErrorAttributes again, it's already a copy right? Maybe the same for `transaction` above.
E.g. `data: transactionErrorAttributes`